lib/Parser/OglExtension.php

```diff
@@ -120,10 +121,56 @@ public function incorporate($other, $commit) {
         }
     }
 
+    /**
+     * Add a sub-extension, or merge it if it already exists.
+     *
+     * @param string $name Name of the extension.
+     * @param string $status Status of the extension.
+     * @param array $supportedDrivers List of drivers supported for this extension.
+     * @param #Mesamatrix\Git\Commit $commit The commit used by the parser.
+     */
+    public function addSubExtension($name, $status, $supportedDrivers = array(), $commit = null) {
+        $newExtension = new OglExtension($name, $status, $this->hints, $supportedDrivers);
+        $subext = $this->findSubExtensionByName($name);
+        if($subext !== null) {
+            $subext->incorporate($newExtension, $commit);
+        }
+        else {
+            $this->subextensions[] = $newExtension;
+        }
+    }
+
+    /**
+     * Get the list of all sub-extensions.
+     *
+     * @return array All the sub-extensions.
+     */
+    public function getSubExtensions() {
+        return $this->subextensions;
+    }
+
+    /**
+     * Find the extensions with the given name.
+     *
+     * @param string $name The name of the extension to find.
+     *
+     * @return \Mesamatrix\OglExtension The extension or null if not found.
```

Comment:
Wrong namespace - should be `\Mesamatrix\Parser\OglExtension`. But since the namespaces are the same, you can use a relative reference by using just `OglExtension`